src/test/java/jssc/junit/rules/DisplayMethodNameRule.java

```diff
@@ -0,0 +1,21 @@
+package jssc.junit.rules;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Rule;
+import org.junit.rules.*;
+import org.junit.runner.*;
+
+/**
+ * Adds the method name to the JUnit logs, useful for debugging
+ */
+public class DisplayMethodNameRule {
+    @Rule
+    public TestWatcher methodWatcher = new TestWatcher() {
+        @Override
+        protected void starting(Description description) {
+            Logger log = LogManager.getLogger(description.getTestClass());
+            log.debug(description.getMethodName());
```

Comment:
I changed this from `log.info` to `log.debug` because I feel that this is better classified as debugging information.  Happy to change it back if preferred. :)

Reply:
At this point it's a matter of taste - which do you prefer. After the changes it would look like this:
DEBUG by default, log method names on DEBUG (verbose):
![Screenshot from 2023-12-10 00-16-16](https://github.com/java-native/jssc/assets/2108093/aa4f3bf2-3f3a-4029-89a2-c207115dad78)

INFO by default, log method names on DEBUG (concise):
![Screenshot from 2023-12-10 00-17-54](https://github.com/java-native/jssc/assets/2108093/152cf577-51ff-49cc-a13b-55b3d572b0d1)

INFO by default, log method names on INFO (in-between):
![Screenshot from 2023-12-10 00-15-28](https://github.com/java-native/jssc/assets/2108093/2f215ac8-2254-4231-91e1-defaa09a5046)

The toggle is easy enough to use, so we can leave it as it is.

Reply:
I think `INFO` looks nicer as it separates the method name from any debug output.